externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/spark/ui/EnginePage.scala

```diff
@@ -58,6 +58,15 @@ case class EnginePage(parent: EngineTab) extends WebUIPage("") {
         <strong>Kyuubi Version: </strong>
         {KYUUBI_VERSION}
       </li>
+      <li>
+        <strong>Compilation Info:</strong>
```

Comment:
How about ?
```
Compilation Revision: ...
Compilation Branch: ...
Compilation with: Spark x.y.z, Hadoop x.y.z ...
```

Reply:
OK, Changed to 'Compilation Revision' and 'Compilation with'.